
Epoch 1
Traceback (most recent call last):
  File "main.py", line 73, in <module>
    ssl_train_losses, model, last_checkpoint = train_scl(model, projector, train_loader, criterion, optimizer, scheduler, dict_config['n_epoch_proj'], use_wandb)
  File "/homes/n20darja/StageCesureCoeur/predict_tavi/ContrastiveLearning/scl.py", line 59, in train_scl
    train_loss = train_epoch_scl(encoder, projector, train_loader, criterion, optimizer, scheduler)
  File "/homes/n20darja/StageCesureCoeur/predict_tavi/ContrastiveLearning/scl.py", line 32, in train_epoch_scl
    data, data_1, data_2 = data_1.to('cuda'), data_2.to('cuda')
ValueError: not enough values to unpack (expected 3, got 2)
Traceback (most recent call last):
  File "main.py", line 73, in <module>
    ssl_train_losses, model, last_checkpoint = train_scl(model, projector, train_loader, criterion, optimizer, scheduler, dict_config['n_epoch_proj'], use_wandb)
  File "/homes/n20darja/StageCesureCoeur/predict_tavi/ContrastiveLearning/scl.py", line 59, in train_scl
    train_loss = train_epoch_scl(encoder, projector, train_loader, criterion, optimizer, scheduler)
  File "/homes/n20darja/StageCesureCoeur/predict_tavi/ContrastiveLearning/scl.py", line 32, in train_epoch_scl
    data, data_1, data_2 = data_1.to('cuda'), data_2.to('cuda')
ValueError: not enough values to unpack (expected 3, got 2)